menu.rb

```diff
@@ -0,0 +1,25 @@
+# Elsa's random menu generator
+# Due: 150511
+# Breakfast + Lunch + 1 Dinner feature: Expand your solution to ensure that no descriptive term in a menu item is ever repeated.
+
+
+# I'm not a foodie, so ha
+food1 = ["spooky", "cumbersome", "alarming", "damp", "furry", "milky", "unnatural", "hissing", "lewd", "jumping", "wiggly"]
+
+food2 = ["brined", "dried", "roasted", "fried", "pickled", "smoked", "sauteed", "steamed", "baked", "seared", "shredded"]
+
+food3 = ["muffins", "lasagne", "muskrat", "liver", "horse", "waffles", "pheasant", "pudding", "trout", "delight", "mystery meat"]
+
+# randomizes food
+random_food1 = food1.shuffle
```

Comment:
Since the order of the original array doesn't really matter in this case - you probably could have just reassigned the values of the food1, food2, food3 variables rather than also creating these randomized ones.